src/main/java/Model/Web/Auth.java

```diff
@@ -33,7 +33,8 @@ public boolean isModelOkayForCreation() {
                 user.getFirstname().equals("") ||
                 user.getLastname().equals("") ||
                 user.getResidence().equals("") ||
-                user.getPhone().equals(""))
+                user.getPhone().equals("") ||
+                (user.getPhone().matches("^(\\+)?[0-9 ]+$") == false))
```

Comment:
tu by bolo dobre pridat komentar k regexu

Reply:
Podpora + na zaciatku pre +421 a tak a zaroven ale podpora pre 00 namiesto +ka s tym, že podpora medzier a iba čisiel

Reply:
je tam slovo phone takze urobil to dostatocne si myslim a je jasne o co ide.
Rozmyslam prave ze je to vlastne konstanta ci by to nebolo dobre vytiahnut na viditelnejsie miesto